src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -1059,4 +1057,54 @@ private void sendEvent(EventType eventType, EventPayload payload, Object caseId)
         .with("transactionId", transactionId)
         .debug("{} event published", eventType);
   }
+
+  private CaseDTO getLatestCaseById(UUID caseId, Boolean getCaseEvents) throws CTPException {
+
+    List<CaseDTO> cases = new ArrayList<>();
+    CaseContainerDTO caseFromRM = null;
+    CaseDTO caseDto = null;
+
+    try {
+      caseFromRM = getCaseFromRm(caseId, getCaseEvents);
+    } catch (ResponseStatusException ex) {
+      if (ex.getStatus() == HttpStatus.NOT_FOUND) {
+        log.with("caseId", caseId).debug("Case Id Not Found by Case Service");
+      } else {
+        log.with("caseId", caseId)
```

Comment:
No code coverage here. Add a unit test.

Reply:
I've now added two:
testRMCaseNotFound()
testRMAndCachedCaseNotFound()

But I'm not sure that the testRMCaseNotFound() one is needed so I can remove it if you like?

Reply:
done